app-benchmarks/stress-ng/files/makefile.patch

```diff
@@ -0,0 +1,75 @@
+--- a/Makefile	2018-04-27 17:02:16.000000000 +0200
++++ b/Makefile	2018-05-05 00:15:00.687399505 +0200
+@@ -21,7 +21,7 @@
+ # Codename "portable pressure producer"
+ #
+ 
+-CFLAGS += -Wall -Wextra -DVERSION='"$(VERSION)"' -O2 -std=gnu99
++CFLAGS += -DVERSION='"$(VERSION)"'
```

Comment:
`-W` flags are fine, actually (except for `-Werror`).

Reply:
Good to know, thanks.